internal/configuration/locations/locations.go

```diff
@@ -35,6 +35,8 @@ var (
 	rallyCorpusDir        = filepath.Join(temporaryDir, "rally_corpus")
 	kubernetesDeployerDir = filepath.Join(deployerDir, "kubernetes")
 	serviceOutputDir      = filepath.Join(temporaryDir, "output")
+
+	serviceSetupDir = filepath.Join(stackDir, "service_setup")
```

Comment:
I'll keep the directory in case it is needed to add any other state.
With the latest changes the file path of this file would be:

```
~/.elastic-package/profiles/<profile>/stack/state/service.json
```